docs/docs/components/README.md

```diff
@@ -0,0 +1,108 @@
+# Overview
+
+- [What are Components?](#what-are-components-)
+- [Component Types](#component-types)
+  * [Sources](#sources)
+  * [Actions](#actions)
+- [Using Components](#using-components)
+- [Developing Components](#developing-components)
+  * [Prerequisites](#prerequisites)
+  * [Quickstart Guides](#quickstart-guides)
+  * [Component API Reference](#component-api-reference)
+- [Sharing Components](#sharing-components)
+  * [Verified Components](#verified-components)
+  * [Community Components](#community-components)
+
+## What are Components?
+
+Components are [Node.js modules](api/#component-structure) that run on Pipedream's serverless infrastructure. They can use Pipedream managed auth for [300+ apps](https://pipedream.com/explore) (for both OAuth and key-based APIs) and [use most npm packages](api/#using-npm-packages) with no `npm install` or `package.json` required. 
+
+Components are most commonly used as the building blocks of Pipedream workflows, but they can also be used like typical serverless functions. You can explore curated components for popular apps in Pipedream's [Marketplace](https://pipedream.com/explore) and [Github repo](https://github.com/pipedreamhq/pipedream/components) or you can author and share your own.
+
+## Component Types
+
+Pipedream supports two types of components — [sources](#sources) and [actions](#actions).
+
+
+### Sources
+
+Sources must be instantiated and they run as independent resources on Pipedream. They are commonly used as workflow triggers (but can also be used as standalone serverless functions).
+
+**Capabilities**
+
+- Accept user input on deploy via `props`
+- [Trigger](api/#interface-props) on HTTP requests, timers, cron schedules, or manually
+- Emit events that can be inspected, trigger Pipedream [workflows](/workflows/) and be consumed in your own app via [API](/api/overview/)
+- Store and retrieve state using the [built-in key-value store](api/#db) 
+- Use any of Pipedream's built-in [deduping strategies](api/#dedupe-strategies)
+- Deploy via Pipedream's UI, CLI or API
+
+**Example**
+
+The Search Mentions source for Twitter is a pre-built component in Pipedream's registry that can be deployed in seconds and emits new Tweets that match a search query. After a user configures and deploys it, the deployed component polls the Twitter API for new results. When new results are found, each new result is emitted as an event that may be inspected and used to trigger a workflow.
```

Comment:
Worth linking to the Search Mentions component?

Reply:
done